arceos-vmm/configs/rk3588-aarch64.toml

```diff
@@ -0,0 +1,40 @@
+id = 1
+name = "linux"
+vm_type = 1
+cpu_set = 2
+cpu_num = 1
+entry_point = 0x1008_0000
+# kernel_path = "linux-aarch64.bin"
+kernel_load_addr = 0x1008_0000
+dtb_load_addr = 0x1000_0000
+
+# The location of image: "memory" | "fs"
+# load from memory
+image_location = "memory" 
+kernel_path = "linux-rk3588-aarch64.bin"
+dtb_path = "linux-rk3588.dtb"
+
+# load from file system
+# image_location = "fs"
+# kernel_path = "arceos-aarch64.bin"
+
+platform = "aarch64-rk3588j"
```

Comment:
why did you add platform related infos in guest VM's config file...

Reply:
The Makefile will read platform
![image](https://github.com/user-attachments/assets/66c37714-4b4f-480f-9be9-8ae95935b4aa)

Reply:
Yep, but this is the platform config for ArceOS, instead of guest VM's config file, 
we should seperate them